protocol/0056-REWA-rewards_overview.md

```diff
@@ -314,6 +336,22 @@ Calculate each entities share of the rewards, $s_{i}$ pro-rata based on $d_{i}$,
 
 $$s_{i} = \frac{d_{i}}{\sum_{i=1}^{n}d_{i}}$$
 
+### Distributing based on lottery
+
+Rewards funded using the lottery-distribution strategy use the same rank-table as described in [0057-TRAN](./0057-TRAN-transfers.md#recurring-transfers-to-reward-accounts) and should be distributed as follows.
+
+1. At the end of the epoch, take the hash of the most recent block as the seed.
+2. Iterate over the rank table, randomly assigning an entity to that rank where an entities probability of selection is as follows.
+
+Let:
+
+- $p_i$ be the probability entity $i$ is selected
+- $r_i$ be the reward score of entity $i$
+
+$$p_{i} = \frac{r_i}{\sum_{i}^{n}{r_i}}$$
```

Comment:
Should be explicit that this is selecting only from the entities that don't already appear in the rank table, i.e. an entity can never appear in the rank table more than once.

Reply:
Should be addressed now.